includes/class-wcpdf-admin.php

```diff
@@ -769,7 +769,7 @@ public function ajax_crud_document() {
 			) );
 		}
 
-		if ( !current_user_can('manage_woocommerce') ) {
+		if ( ! apply_filters( 'wpo_wcpdf_current_user_is_allowed', current_user_can( 'manage_woocommerce' ), sanitize_text_field( $_POST['document_type'] ) ) ) {
```

Comment:
This should be applied here too, too keep everything the same, admin privileges wise:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/f61d715cf49d0745c90fdb81eb4956763d462107/includes/class-wcpdf-main.php#L319

Reply:
I have restructured the permissions check somewhat in aa90c8089ef2b3ff79ac14fc44b9aadfb37fd058. It should do exactly the same as before (before this PR), except that the code is now a bit less opaque. Please double check because it's easy to overlook errors here. If you confirm it still does the same thing, you can apply your filters there too.

Reply:
Do we need the `wpo_wcpdf_current_user_is_allowed` filter here as we already have the `wpo_wcpdf_check_privs` filter?

Reply:
The goal is to have one filter to rule them all, so: yes we need it here too :)

Reply:
Added `wpo_wcpdf_current_user_is_allowed` filter to `generate_pdf_ajax()` here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/commit/a435c0ef36e31f8a38c128b654bd76f50b18aaf9

Reply:
you forgot the other change (`( current_user_can( 'manage_woocommerce_orders' ) || current_user_can( 'edit_shop_orders' )`) mentioned above:


> This should be applied here too, too keep everything the same, admin privileges wise:
> 
> https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/f61d715cf49d0745c90fdb81eb4956763d462107/includes/class-wcpdf-main.php#L319

Reply:
Changed here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/356/commits/32a08bd2bbd98c02691ae1c21cd68a93467d118e